blockchain/state/statedb.go

```diff
@@ -718,8 +718,8 @@ func (self *StateDB) createObjectWithMap(addr common.Address, accountType accoun
 // CreateAccount is called during the EVM CREATE operation. The situation might arise that
 // a contract does the following:
 //
-//   1. sends funds to sha(account ++ (nonce + 1))
-//   2. tx_create(sha(account ++ nonce)) (note that this gets the address of 1)
+//  1. sends funds to sha(account ++ (nonce + 1))
+//  2. tx_create(sha(account ++ nonce)) (note that this gets the address of 1)
```

Comment:
I think the comment is outdated now. Can you update the comment with the following contents?
```
CreateAccount is currently used for test code only. Use CreateEOA, CreateSmartContractAccount, or CreateSmartContractAccountWithKey instead to create an explicitly typed account. 
```